### ISSUE TITLE
ADE Finding: ITF caution icon

### ISSUE THREAD

**Issue by zslocum1** [department-of-veterans-affairs/va.gov-team#138340]
## Background/Context

During team qa, the ADE team found that there's a "caution" icon next to the ITF date entry that doesn't have an accessible name / isn't announced by screen readers. It conveys meaning ("warning") and SR users need to have access to that.

## Examples

<img width="155" height="67" alt="Image" src="https://github.com/user-attachments/assets/70ce678d-4804-4856-a520-23432412bde6" />
